test/integration/short/geometry/polygon-tests.js

```diff
@@ -130,7 +130,12 @@ vdescribe('dse-5.0', 'Polygon @SERVER_API', function () {
                     normalizedCoordinates.push([c[0], c[2], c[1], c[3]]);
                   }
                 }
-                assert.deepEqual(normalizedCoordinates, polygon.toJSON().coordinates);
+                if (helper.isDseGreaterThan('6.9')) {
+                  //TODO: find documentation
+                  assert.deepEqual(value, polygon.toJSON());
+                }else{
+                  assert.deepEqual(normalizedCoordinates, polygon.toJSON().coordinates);
+                }
```

Comment:
Change to address NODEJS-675.

Worth noting that there is some [commentary](https://github.com/datastax/nodejs-driver/blob/v4.7.2/test/integration/short/geometry/polygon-tests.js#L121-L124) in that test about why this normalization is occurring.  Might be worth mentioning to the DSE team that we noticed this change and check with them about whether this was a deliberate modification on their part or an unintended side effect.

Reply:
Worth noting: @SiyaoIsHiding has asked this question internally but we haven't heard a response back.

Reply:
Created JIRA ticket: https://datastax.jira.com/browse/DSP-24515